index.js

```diff
@@ -133,11 +133,13 @@ var createAssertion = function createAssertion(type, message, wrappers, block, m
 	if (descriptors.length === 0) {
 		throw new RangeError(inspect(type) + ' called with no wrappers defined');
 	}
-	var describeMsgs = [];
-	forEach(wrappers, function (wrapper) {
-		checkThis(wrapper);
-		describeMsgs.push(getThisDescription(wrapper));
```

Comment:
`concatThis` copies the wrappers array, which is where the descriptors are stored so they should get copied over.

We're also already mutating the descriptor object by turning each of its key/values into arrays. https://github.com/ljharb/jest-wrap/blob/master/index.js#L220

We could probably shrink the api here a bit by changing `.extend` to just take a descriptor, and .`use` to take a function that returns a descriptor, then validating that descriptor.